vars/govuk.groovy

```diff
@@ -213,6 +213,9 @@ def nonDockerBuildTasks(options, jobName, repoName) {
         testGemWithAllRubies(extraRubyVersions)
       } else {
         stage("Run tests") {
+          if (hasAssets() && options.precompileAssetsBeforeTests) {
+            precompileAssets("test")
```

Comment:
Would this mean that most of the time assets will be compiled twice? Would it work if we put the existing `precompileAssets` call before the test run?

Reply:
Yeah I guess they would be precompiled twice. When we use Capybara, I think this happens already because it triggers that's what it does.

If we precompile with a `RAILS_ENV=production` will they be used during the test steps when presumably rails runs with `RAILS_ENV=test`?

Reply:
Not sure, perhaps the we could always run it with `RAILS_ENV=test`. The purpose of the original `precompileAssets` is to make sure that there aren't any syntax errors in the CSS, which this would cover as well.

Reply:
👍 thanks Tijmen - updated approach 👉 https://github.com/alphagov/govuk-jenkinslib/pull/50